packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -543,6 +548,17 @@ public void requestContactCustomerSupport() {
             public void requestGotoCustomerSupportOptions() {
                 mOnCustomerSupportOptionsListener.onGotoCustomerSupportOptions();
             }
+
+            @Override
+            public void sendEventToHost(String eventName,  ReadableMap properties) {
+                Map<String, Object> propertiesMap = new HashMap<>();
+                ReadableMapKeySetIterator iterator = properties.keySetIterator();
+                while (iterator.hasNextKey()) {
+                    String key = iterator.nextKey();
+                    propertiesMap.put(key, properties.getDynamic(key));
```

Comment:
Turns out `ReadableMap.getDynamic()` returns a `Dynamic` object which barfs when calling `JSONObject(properties)` [like here for example](https://github.com/wordpress-mobile/WordPress-Analytics-Android/blob/develop/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java#L514). Everything ends up as `null`

I added a switch case on the type:
```java
switch (properties.getType(key)) {
   case Null:
       break;
   case Boolean:
     propertiesMap.put(key, properties.getBoolean(key));
     break;
   case Number:
       propertiesMap.put(key, properties.getDouble(key));
       break;
   case String:
      propertiesMap.put(key, properties.getString(key));
       break;
    case Map:
         propertiesMap.put(key, properties.getMap(key));
         break;
    case Array:
         propertiesMap.put(key, properties.getArray(key));
         break;
}
```

That seems to work fine on the `blockInserted` event but I noticed an empty array still end up as `null`, which is not great.